src/Tests/Microsoft.NET.PerformanceTests/Microsoft.NET.PerformanceTests.csproj

```diff
@@ -40,4 +40,16 @@
 
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
 
+  <Target Name="PopulatePerformanceTestsFromGitHub" BeforeTargets="CoreCompile">
+    <PropertyGroup>
+      <PerformanceTestsRepoURL>https://github.com/dotnet/BuildPerformanceTestAssets</PerformanceTestsRepoURL>
+      <PerformanceTestsCommit>daf705c</PerformanceTestsCommit>
+      <PerformanceTestsLocalDirectory>$(RepoRoot).perftestsource</PerformanceTestsLocalDirectory>
+    </PropertyGroup>
+
+    <Exec Condition="!Exists($(PerformanceTestsLocalDirectory))"
+          WorkingDirectory="$(RepoRoot)" Command="git clone --quiet $(PerformanceTestsRepoURL) $(PerformanceTestsLocalDirectory)" />
+    <Exec WorkingDirectory="$(PerformanceTestsLocalDirectory)" Command="git checkout --quiet $(PerformanceTestsCommit)" />
```

Comment:
I would prefer to do the git clone in C# instead of MSBuild.

Reply:
@dsplaisted 
Does git have C# apis?